tsconfig.json

```diff
@@ -0,0 +1,27 @@
+{
+  "compilerOptions": {
+    "allowJs": true,
+    "allowSyntheticDefaultImports": true,
+    "declaration": true,
+    "esModuleInterop": true,
+    "forceConsistentCasingInFileNames": true,
+    "jsx": "react",
+    "lib": ["dom", "es6", "es2019"],
```

Comment:
I don't think we need to keep `es2019` in here, since we're saying we really want to transpile down to ES6 here.

Also, one of the questions posed in the Paragon tsconfig [PR](https://github.com/openedx/paragon/pull/1962/files#r1088764652) is whether we need `dom.iterable`; it doesn't automatically get included via `dom` here. I believe we're going to opt to include it, but curious to hear your thoughts here.

Reply:
Went with dom.iterable, since it's not included with dom, and seems like something we'd use at some point if we're not already.